examples/battetl_demo.ipynb

```diff
@@ -62,8 +62,7 @@
     }
    ],
    "source": [
-    "#battetl_dir = \"YOUR/BATTETL/DIRECTORY/PATH\"\n",
-    "battetl_dir = '/Users/zander/Documents/Work/BattGenie/battetl'\n",
+    "battetl_dir = \"YOUR/BATTETL/DIRECTORY/PATH\"\n",
```

Comment:
@SyXuan @tanevitt why do we need the battetl path? If somepone pip installed battetl, they may not easily know what the path is. In that case, if we do need it, we should also show a generic way to get the path, or automatically get it with some `os.path()` jugglery ourselves from the system.